src/include/cache.h

```diff
@@ -13,7 +13,7 @@
 #define	WT_EVICT_INT_SKEW  (1<<20)	/* Prefer leaf pages over internal
 					   pages by this many increments of the
 					   read generation. */
-#define	WT_EVICT_WALK_PER_FILE	 10	/* Pages to queue per file */
+#define	WT_EVICT_WALK_PER_FILE	 50	/* Pages to queue per file */
```

Comment:
Over a number of runs with the current logic there isn't much noticeable speed up.

I've reverted this part of the change.